packages/palette_generator/lib/palette_generator.dart

```diff
@@ -1162,7 +1162,7 @@ class _ColorCutQuantizer {
   }
 
   // Iterate through the [PriorityQueue], popping [_ColorVolumeBox] objects
-  // from the queue and splitting them. Once split, the new box and the
+  // from the queue and splitting them. Once split, thebox and the
```

Comment:
missing space

Reply:
Wow.  This file was not meant to be changed.